test/playground.test.ts

```diff
@@ -5,19 +5,13 @@ import {Tab} from '../src/sdk';
 import {PLAYGROUND_ORIGIN} from '../src/constants';
 import {PluginSdk} from '../src/plugin';
 
-jest.mock('../src/constants', () => ({
-    PLAYGROUND_CONNECT_URL: 'https://play.croct.com/connect.html',
-    PLAYGROUND_ORIGIN: 'https://play.croct.com',
-}));
-
-beforeEach(() => {
-    sessionStorage.clear();
-    window.history.replaceState({}, 'Home page', 'http://localhost');
-});
-
-afterEach(() => {
-    jest.restoreAllMocks();
-});
```

Comment:
Can you keep it like that?? So you don't repeat it.

Reply:
Jest hooks shouldn't be inside each describe?

Reply:
It can be outside. Using it inside just limits the scope to only run in that describe.